src/lib.rs

```diff
@@ -1835,3 +1834,18 @@ pub(crate) fn total_anchor_channels_reserve_sats(
 			* anchor_channels_config.per_channel_reserve_sats
 	})
 }
+
+fn setup_tcp_listener(addr: SocketAddr) -> Result<TcpListener, tokio::io::Error> {
+	let socket = match addr {
+		SocketAddr::V4(_) => TcpSocket::new_v4()?,
+		SocketAddr::V6(_) => TcpSocket::new_v6()?,
+	};
+	socket.set_reuseaddr(true)?;
+	#[cfg(not(target_os = "windows"))]
+	socket.set_reuseport(true)?;
```

Comment:
Isn't this a load balancing option, where the kernel distributes load to any of the process that bind to the same port? Not sure if that is desirable here?